packages/snap/src/keyring.test.ts

```diff
@@ -390,9 +390,12 @@ describe('Keyring', () => {
 
   describe('Delete Account', () => {
     it('should delete an account', async () => {
-      const account = await keyring.createAccount({ privateKey: aaOwnerSk });
+      const account = await keyring.createAccount({
+        privateKey: aaOwnerSk,
+      });
       await keyring.deleteAccount(account.id);
       await expect(keyring.getAccount(account.id)).rejects.toThrow(
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
nit but I think adding this to the tsconfig would mean you don't need to add a comment here to disable 
```json
"types": [
    {
      "Account": {
        "id": "string"
      }
    }
  ]
  ```